src/javascript/_common/remote_config.json

```diff
@@ -1 +1 @@
-{"cs_chat_livechat":true,"cs_chat_whatsapp":true,"marketing_growthbook":true,"passkeys":true,"tracking_GTM":true,"tracking_datadog":false,"tracking_hotjar":true,"tracking_rudderstack":true}
+{"cs_chat_livechat":true,"cs_chat_whatsapp":false,"hub_enabled_country_list":["bs","tw","rs","am","pg","gn","al","gd","sr","gy","mk","td","is","gm","ad","cv","mr","bz","sc","cf","aq","dz","tm","gp","kn","lc","tl","fj","gq","bt","km","mc","pw","sb","gl","dm","sv","ge","kw","kg","ax","vg","aw","re","cw","mq","gf","pf","ms","me","ai","bm","tc","mo","io","nu","yt","sx","cn","va","mf","ki","gi","er","tk","st","sm","wf","cx","nf","cc","to","fo","eh","mh","sh","pn","fm","bl","fk","li","nr","pm"],"marketing_growthbook":true,"passkeys":true,"tracking_GTM":true,"tracking_datadog":true,"tracking_hotjar":true,"tracking_rudderstack":true}
```

Comment:
@akmal-deriv does this mean we need to keep updating the `country_list` every time it's updated at remote? Why are we keeping a hard coded list in FE?